tape/src/main/java/com/squareup/tape2/FileObjectQueue.java

```diff
@@ -94,7 +96,8 @@
       try {
         return converter.from(data);
       } catch (IOException e) {
-        throw new RuntimeException("todo: throw a proper error", e);
+        sneakyRethrow(e);
```

Comment:
If you want, you can make this return `Throwable` so that you do `throw sneakyRethrow(e)` which saves you from dealing with control flow problems after calling this method.

Reply:
can i? the tricky thing is that Throwable is a caught exception, so i can't `throw getSneakyThrowable(ioe)` in the iterator methods that don't declare the exception.

Reply:
Use RuntimeException then

Reply:
i think that'll then throw a class cast exception because it actually does the cast to RuntimeException.
i'll try it. brb.

Reply:
the method never returns

Reply:
right. thanks!